accessors/spanner/spanner_accessor.go

```diff
@@ -47,6 +67,18 @@ type SpannerAccessor interface {
 	ValidateChangeStreamOptions(ctx context.Context, changeStreamName, dbURI string) error
 	// Create a change stream with default options.
 	CreateChangeStream(ctx context.Context, adminClient spanneradmin.AdminClient, changeStreamName, dbURI string) error
+	// Create new Database using conv
+	CreateDatabase(ctx context.Context, adminClient spanneradmin.AdminClient, dbURI string, conv *internal.Conv, driver string, migrationType string) error 
+	// Update Database using conv
+	UpdateDatabase(ctx context.Context, adminClient spanneradmin.AdminClient, dbURI string, conv *internal.Conv, driver string) error
+	// Updates an existing Spanner database or create a new one if one does not exist using Conv
+	CreateOrUpdateDatabase(ctx context.Context, adminClient spanneradmin.AdminClient, dbURI, driver string, conv *internal.Conv, migrationType string) error 
+	// Check whether the db exists and if it does, verify if the schema is what we currently support.
+	VerifyDb(ctx context.Context, adminClient spanneradmin.AdminClient, dbURI string) (dbExists bool, err error)
```

Comment:
Pls add unit tests for verifydb and validateddl as well.

Reply:
I am not mocking verifyDb and validateDdl so the LOC are covered and tested. i will add error scenarios as well so not line is left untested. Is that okay?

Reply:
Yeah that should be fine as long as we get coverage.